packages/trader/src/Modules/Trading/Components/Elements/mobile-widget.jsx

```diff
@@ -1,28 +1,17 @@
 import React from 'react';
 import { Money } from '@deriv/components';
-import { localize, Localize } from '@deriv/translations';
+import { localize } from '@deriv/translations';
 import { getExpiryType, getDurationMinMaxValues, getLocalizedBasis } from '@deriv/shared';
```

Comment:
Tests for mobile-widget will be added after the 6th Package of TS migration will me merged. Card for tests: WEBREL-64